oak_functions/load_test/src/main.rs

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use anyhow::Context;
+use bencher::stats::Stats;
+use std::time::Instant;
+
+// From https://pantheon.corp.google.com/api-gateway/gateway/weather-lookup-grpc/location/europe-west2?project=oak-ci.
+const URL: &str = "https://weather-lookup-grpc-8tk01hn7.nw.gateway.dev";
+const REQUEST: &[u8] = br#"{"lat":52,"lon":0}"#;
+const TOTAL_REQUESTS: usize = 50;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let mut client =
+        oak_functions_client::Client::new(&URL.parse().context("Could not parse URL")?)
+            .await
+            .context("Could not create client")?;
+
+    let mut latencies_millis = Vec::<f64>::with_capacity(TOTAL_REQUESTS);
+
+    for i in 0..TOTAL_REQUESTS {
+        let start = Instant::now();
+        let response = client
+            .invoke(REQUEST)
```

Comment:
Not sure how repeatedly sending the same request affects the measurements. But I think it should be fine for now.

Reply:
We don't do any caching, so it should be pretty realistic (modulo internal CPU caches)